lighthouse-cli/test/smokehouse/test-definitions/core-tests.js

```diff
@@ -65,7 +65,7 @@ const smokeTests = [{
 }, {
   id: 'metrics',
   expectations: require('./tricky-metrics/expectations.js'),
-  config: require('../../../../lighthouse-core/config/perf-config.js'),
+  config: require('../../../../lighthouse-core/config/no-throttling-config.js'),
```

Comment:
maybe it would be good to document why this config is being used here for future reference? e.g. I'm pretty sure we only used `perf-config` here because it was a convenient, preexisting config to check out the perf category with devtools throttling, but if there were more specific tricky reasons as well, I would have long forgotten them by now :)

(I assume from the comment in `tricky-tti-late-fcp.html` the test would fail if switching to a different throttling, but that might not always be the case)

Reply:
> maybe it would be good to document why this config is being used here for future reference?

Or would that comment make more sense in the expectations file instead? Since it would allow a comment per test url, but authors are still likely to look there if changing things (whereas they might not look at comments in the html fixture until a test actually starts failing).

Don't throw things at me if this is yet another thing your smokehouse refactor would have made easy 😬